.github/scripts/create_pr_for_staged_changes.sh

```diff
@@ -0,0 +1,114 @@
+#!/bin/bash
+set -uxo pipefail # enable debugging, prevent accessing unset env vars, prevent masking pipeline errors to the next command
+
+#docs
+#title              :create_pr_for_staged_changes.sh
+#description        :This script will create a PR for staged changes and detect and close duplicate PRs.
+#author		        :@heitorlessa
+#date               :May 8th 2023
+#version            :0.1
+#usage		        :bash create_pr_for_staged_changes.sh {git_staged_files_or_directories_separated_by_space}
```

Comment:
```suggestion
#author		    :@heitorlessa
#date               :May 8th 2023
#version            :0.1
#usage		    :bash create_pr_for_staged_changes.sh {git_staged_files_or_directories_separated_by_space}
```

this is killing my OCD hahaha
![image](https://user-images.githubusercontent.com/4295173/236819502-ebf520bf-451e-4272-ab82-09bad78ee115.png)

Reply:
ooh must've been IDE formatting that added some spaces HAHAHA because this is what it looks to me

![Uploading image.png…]()